### PR TITLE
tools_gradle_intellij_plugin.md: Contributing section

### PR DESCRIPTION
**Pull request by hsz**

### CODE REVIEW

File: topics/appendix/tools/tools_gradle_intellij_plugin.md [hunk 1]
```diff
@@ -110,6 +110,64 @@ pluginManagement {
...
 
+## Contributing
+There are many ways to contribute to the Gradle IntelliJ Plugin project, and each of them is valuable to us.
+Every submitted feedback, issue, or pull request is highly appreciated.
+
+### Issue Tracker
+The Gradle IntelliJ Plugin is an Open-Source project hosted on GitHub: [JetBrains/gradle-intellij-plugin](https://github.com/JetBrains/gradle-intellij-plugin).
+All project sources, discussions, and issues are publicly available, as we rely on the [GitHub Issues](https://github.com/JetBrains/gradle-intellij-plugin/issues) to track any feature requests or reported problems.
+
+Before reporting an issue, please update your configuration to use always the [latest release](#usage), or try with the [snapshot release](#snapshot-release), which contains not-yet publicly available changes.
```
Comment (YannCebron) [inline topics/appendix/tools/tools_gradle_intellij_plugin.md:121]: also suggest using latest Gradle version and/or upgrading development IDE version

File: topics/appendix/tools/tools_gradle_intellij_plugin.md [hunk 2]
```diff
@@ -110,6 +110,64 @@ pluginManagement {
...
+- full stack trace
+- steps to reproduce it
+- plugin version you use
+- your build configuration
+- ideally, the minimal project will let us reproduce the issue locally.
+
+### Building Project
+If you want to run the Gradle IntelliJ Plugin directly from sources, i.e., to test it against the latest changes, debug the plugin, or verify your changes before creating a pull request, clone it from our GitHub repository:
+
+{style="narrow"}
```
Comment (YannCebron) [inline topics/appendix/tools/tools_gradle_intellij_plugin.md:134]: why do we need to document this explicitly?

**Review by YannCebron — COMMENTED** [review]
I'd rather put this on a separate page or at least not on the top of this page. we can link with short notice about how to submit bugs/development on the main user-facing page instead.

File: topics/appendix/tools/tools_gradle_intellij_plugin.md [hunk 3]
```diff
@@ -110,6 +110,64 @@ pluginManagement {
...
+HTTPS
+: https://github.com/JetBrains/gradle-intellij-plugin.git
+
+SSH
+: git@github.com:JetBrains/gradle-intellij-plugin.git
+
+GitHub CLI
+: gh repo clone JetBrains/gradle-intellij-plugin
+
+There are three possible ways to run the Gradle IntelliJ Plugin using its sources:
```
Comment (karollewandowski) [inline topics/appendix/tools/tools_gradle_intellij_plugin.md:144]: I think that after using a colon-ending sentence it should be a list or it should be just a regular sentence.

File: topics/appendix/tools/tools_gradle_intellij_plugin.md [hunk 4]
```diff
@@ -110,6 +110,64 @@ pluginManagement {
...
+It is possible to link your project with the Gradle IntelliJ Plugin project, so it'll be loaded and built as a module.
+To integrate it with another consumer-like project, in the <path>settings.gradle.kts</path> file (or <path>settings.gradle</path> for Groovy), add the following line and refresh your Gradle configuration:
+
+```kotlin
+includeBuild("/path/to/gradle-intellij-plugin")
+```
+
+### Pull Requests
+To correctly prepare the pull requests, make sure to provide the following information:
+- proper title and description of the GitHub Pull Request – describe what your change introduces, what issue does it fix
```
Comment (karollewandowski) [inline topics/appendix/tools/tools_gradle_intellij_plugin.md:165]:
```suggestion
- proper title and description of the GitHub Pull Request – describe what your change introduces, what issue it fixes, etc.
```

**Comment by YannCebron** [comment]
alternatively, most of these contents could live in CONTRIBUTING.md inside the gradle plugin repo itself (it should be added anyway and link to this page)

**Pull request closed without merging**